src/Metadata/Property/Factory/CachedPropertyMetadataFactory.php

```diff
@@ -28,6 +28,7 @@
 
     private $cacheItemPool;
     private $decorated;
+    private $memoryCache = [];
```

Comment:
APCu is also a (shared) memory cache. This is the local memory cache. :laughing: